tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -285,3 +286,15 @@ Check Cards Details Are Correct
         Check Card Image  card_locator=${card_xp}  app_id=${application_id}  expected_data=${expected_data}
         Check Get Started Sidebar  card_locator=${card_xp}  card_badges=${badges_titles}  app_id=${application_id}  expected_data=${expected_data}
     END
+
+Check CSS Property Has The Expected Value
+    [Documentation]     Compare the actual CSS property value with the expected one
+    [Arguments]   ${locator}    ${property}    ${exp_value}   ${operation}=equal
+    ${el_text}=   Get Text   xpath:${locator}
```

Comment:
Do we really need this ?

Reply:
I thought it could be useful during debugging, to understand what part of the text the robot is reading

Reply:
IMHO if you want to leave it in you should also log the value

Reply:
you can read the value from the RF logs without explicitly log it, I preferred to spare a keyword, but if you think it is better I will do it

Reply:
yeah we can read it but if output is big in text it doesn't show all content if i remember correctly

Reply:
yeah, but you don't need to read a such big content. it just help you understand what you need to look on the dashboard

Reply:
IMHO if we capture it we should also log it; We won't need to use it most of the times but if there's a failure we know that we will able to access all information

Reply:
okay, I've added the log

Reply:
Thanks!